pyperf/_bench.py

```diff
@@ -424,6 +424,39 @@ def median_abs_dev(self):
             raise ValueError("MAD must be >= 0")
         return value
 
+    def required_nsamples(self):
+        """
+        Determines the number of samples that would be required to have 95%
+        certainty that the samples have a variance of less than 1%.
+
+        This is described in this Wikipedia article about estimating the sampling of
+        a mean:
+
+        https://en.wikipedia.org/wiki/Sample_size_determination#Estimation_of_a_mean
+        """
+        # Get the means of the values per run
```

Comment:
Why not computing the mean only once, for all values of all runs?

Reply:
Because for some benchmarks, cache effects are visible within the same process.  For example, pylint takes about 30% longer during the first iteration than the subsequent 2 iterations.  One could argue that's a bad benchmark, but it's common enough that we should control for it.  There's some more discussion here: https://github.com/faster-cpython/bench_runner/issues/318#issuecomment-2537260960

That said, it's definitely worth putting a comment about that here.